app/src/main/kotlin/jp/co/yumemi/android/code_check/RepositorySearchViewModel.kt

```
@@ -107,4 +112,15 @@ class RepositorySearchViewModel(application: Application) : AndroidViewModel(app
             )
         }
     }
+
+    fun searchRepositories(query: String) {
+        viewModelScope.launch {
+            try {
+                val results = fetchSearchResults(query)
+                _searchResults.postValue(results)
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)